modules/Bio/EnsEMBL/Variation/DBSQL/PhenotypeFeatureAdaptor.pm

```diff
@@ -1074,6 +1074,92 @@ sub fetch_all {
   
 }
 
+sub get_PhenotypeFeatures_by_location {
+  my $self = shift;
+  my $seq_region_id = shift;
+  my $seq_region_start = shift;
+  my $seq_region_end = shift;
+
+  throw("Cannot fetch attributes without dbID") unless defined($seq_region_id);
```

Comment:
Could we replace "Cannot fetch attributes without dbID" by something like "Cannot fetch attributes without location information" ?